src/table/utils/handle-key-press.ts

```diff
@@ -76,26 +102,41 @@ export const arrowKeysNavigation = (evt, rowAndColumnCount, cellCoord, topAllowe
   return [nextRow, nextCol];
 };
 
-export const getRowAndColumnCount = (rootElement) => {
-  const rowCount = rootElement.getElementsByClassName('sn-table-row').length;
-  const columnCount = rootElement.getElementsByClassName('sn-table-head-cell').length;
-
-  return { rowCount, columnCount };
-};
-
-export const moveFocus = (evt, rootElement, cellCoord, setFocusedCellCoord, topAllowedRow) => {
+/**
+ * resets and adds new focus to a table cell based which key is pressed
+ */
+export const moveFocus = (
+  evt: React.KeyboardEvent,
+  rootElement: HTMLElement,
+  cellCoord: [number, number],
+  setFocusedCellCoord: React.Dispatch<React.SetStateAction<[number, number]>>,
```

Comment:
same as changeSortOrder. We can extract it to be SetFocusedCellCoord.

Reply:
yeah there are a bunch of these. Since they are so long, it could be good to extract them

Reply:
hmm I think it is better to do in a different PR though

Reply:
sure